src/Keboola/S3Extractor/Finder.php

```diff
@@ -157,26 +157,26 @@ private function listFilesInPrefix(): \Iterator
                 }
 
                 $file = new File($this->bucket, $object['Key'], $object['LastModified'], (int)$object['Size'], $dst);
-                $filesToDownloadCount++;
+                $filesMatchedCount++;
 
-                if ($this->isFileOld($file)) {
-                    continue;
-                }
-                $newFilesCount++;
-
-                yield $file;
-
-                $isImportantMilestoneForListed = ($filesListedCount % 10000) === 0
-                    && $filesListedCount !== 0;
-                $isImportantMilestoneForDownloaded = ($filesToDownloadCount % 1000) === 0
-                    && $filesToDownloadCount !== 0;
-                if ($isImportantMilestoneForListed || $isImportantMilestoneForDownloaded) {
+                // log progress
+                if ($filesListedCount !== 0 &&
+                    $filesMatchedCount !== 0 &&
+                    (($filesListedCount % 10000) === 0 || ($filesMatchedCount % 1000) === 0)
+                ) {
```

Comment:
Uprava podmienky.

`$filesToDownloadCount` premenované na `$filesMatchedCount`, kedze to bolo zavadzajuce.

`$filesToDownloadCount` este nemajú vyhodnotené `newFilesOnly` a `limit`.